src/pip/_internal/wheel.py

```diff
@@ -71,7 +71,7 @@ def open_for_csv(name, mode, encoding='utf8'):
     else:
         nl = {'newline': ''}
         bin = ''
-    return open(name, mode + bin, **nl)
+    return open(name, mode + bin, encoding=encoding, **nl)
```

Comment:
Again, you really need to add a test: what about Python 2 support (the file is opened in binary mode)?